internal/collect_es6_sources.bzl

```diff
@@ -40,11 +40,11 @@ def collect_es6_sources(ctx):
     if file.short_path.startswith(".."): continue
 
     rerooted_file = ctx.actions.declare_file(
-      "%s.es6/%s" % (
-        ctx.label.name,
+      "%s/%s" % (
```

Comment:
is it required to remove the suffix? What happens if we need a second re-rooted output under the label?

Reply:
Done. <label_name>.es6 is back in but needed to add in the short_path before the basename.